README.md

```diff
@@ -1463,24 +1463,25 @@ events, and will print out only status records.
 
 A typical use case would be the following:
 ```lua
+tnt = require 'torchnet'
 
 -- require the viewers we want
-local logtext = require 'torchnet.log.view.text'
-local logstatus = require 'torchnet.log.view.status'
+logtext = require 'torchnet.log.view.text'
+logstatus = require 'torchnet.log.view.status'
 
-local log = tnt.Log{
+log = tnt.Log{
```

Comment:
Why remove all the `local`s here. I would encourage anyone to *never* use global variables. Ever.